strawberry_django/utils/query.py

```diff
@@ -101,7 +101,7 @@ def filter_for_user_q(
         except KeyError:  # pragma:nocover
             # If we are not running async, retrieve it
             ctype = (
-                ContentType.objects.get_for_model(model)
+                ContentType.objects.get_for_model(model, for_concrete_model=False)
```

Comment:
I'm not very used to proxy models, but does this have any bad side effect for non-proxy ones?

Reply:
Shouldn't have any effect.  It merely allows for fetching of proxy-model content-types.  https://docs.djangoproject.com/en/4.2/ref/contrib/contenttypes/#django.contrib.contenttypes.models.ContentTypeManager.get_for_model